### PR TITLE
[doc]update and delete

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
add keyword `UPDATE` and `DELETE`

### CODE REVIEW

File: docs/manual-CN/statement-syntax/data-manipulation-statements/update-syntax.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Update Syntax
+Nebula 支持 `UPDATE` 一个点或者一条边的属性，支持CAS操作，支持返回相关的属性。
+## Update vertex
+
+```
+UPDATE VERTEX $vid SET $update_columns WHEN $condition YIELD $columns
```
Comment (whitewum) [inline docs/manual-CN/statement-syntax/data-manipulation-statements/update-syntax.md]: 这个语法形式和其他几个文档的语法形式不太一致啊。
$abc  <abc>

File: docs/manual-CN/statement-syntax/data-manipulation-statements/update-syntax.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+- `$update_columns` 表示需要更新的 tag 上的 columns，比如 `tag1.col1 = $^.tag2.col2 + 1`表示把这个点的 `tag1.col1` 更新成 `tag2.col2 + 1`。
+
+    **注意：**  `$^`表示 `UPDATE` 中表示需要更新的点
+
+- `$condition` 是一些约束条件，只有满足这个条件，`UPDATE` 才会真正执行，支持表达式操作。
+- `$columns` 表示需要返回的columns，此处 `YIELD` 可返回update以后最新的columns值。
+
+举一个例子：
+```
+UPDATE VERTEX 101 SET course.credits = $^.course.credits + 1, building.name = "No8" WHEN $^.course.name == "Math" && $^.course.credits > 2 YIELD $^.course.name AS Name, $^.course.credits AS Credits, $^.building.name
```
Comment (whitewum) [inline docs/manual-CN/statement-syntax/data-manipulation-statements/update-syntax.md]: course.credits = $^.course.credits + 1
这个没错吗？

Comment (whitewum) [inline docs/manual-CN/statement-syntax/data-manipulation-statements/update-syntax.md]: 举例的话，我一般会 
nebula>xxxxx

File: docs/manual-CN/statement-syntax/data-manipulation-statements/delete-syntax.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Delete Syntax
+Nebula 支持给定一个 vertex Id，删掉这个顶点和与它相关联的入边和出边，语法如下：
+
+```
+DELETE VERTEX $vid
+```
+系统内部会找出这个与顶点相关联的出边和入边，把他们全部删除成功后，最后删除点相关的信息。整个过程当前还无法保证原子性。
```
Comment (whitewum) [inline docs/manual-CN/statement-syntax/data-manipulation-statements/delete-syntax.md]: 整个过程当前还无法保证原子性，因此当遇到失败时请重试。

File: docs/manual-CN/statement-syntax/data-manipulation-statements/update-syntax.md (hunk 2 above)
Comment (dangleptr) [reply docs/manual-CN/statement-syntax/data-manipulation-statements/update-syntax.md]: It's right.  $^ can't be ignored.

Comment (dangleptr) [reply docs/manual-CN/statement-syntax/data-manipulation-statements/update-syntax.md]: Make sense.  Would you mind give a new example for update/delete   @Amber1990Zhang

File: docs/manual-CN/statement-syntax/data-manipulation-statements/update-syntax.md (hunk 1 above)
Comment (dangleptr) [reply docs/manual-CN/statement-syntax/data-manipulation-statements/update-syntax.md]: Never mind.  It's just a symbol.

File: docs/manual-CN/statement-syntax/data-manipulation-statements/update-syntax.md (hunk 2 above)
Comment (Amber1990Zhang) [reply docs/manual-CN/statement-syntax/data-manipulation-statements/update-syntax.md]: Sure, I'll take care of that.

**Pull request merged**